scalyr_agent/tests/configuration_test.py

```diff
@@ -1423,11 +1423,16 @@ def convert_path(self, path):
         return self.make_path(None, path)
 
 
+class NullHandler(logging.Handler):
```

Comment:
It looks like `FakeAgentLogger` is identical to the one in `scalyr_agent/test_util.py`.  Can we just import that one rather than defining it again (same then will `NullHandler` and `FakePlatform`)

Reply:
Yup, I'm aware of the code duplication . (This is from an earlier PR. There was some reason why I chose to leave it in there for the time being but with intention to deduplicate.  I'll deduplicate in a different PR . (There's actually more stuff that needs fixing/cleaning up especially pertaining to python backwards compatibility.  For example with context manager usage.  I should try to keep all that separate from this PR and have this PR be mainly about the CircleCI config.)

Reply:
Sounds good.